docs/includes/word-preview-apis-note.md

```diff
@@ -0,0 +1,5 @@
+> [!IMPORTANT]
+> Note that the following Word preview APIs are only available on the following platforms.
+> - Office on Windows
```

Comment:
Might as well be explicit about the platform (so we don't have to worry about branding differences between Office and M365).
```suggestion
> - Word on Windows
```